File: member-service/src/main/java/com/hedvig/memberservice/services/member/dto/MemberSignResponse.java
package com.hedvig.memberservice.services.member.dto;

import com.hedvig.external.bankID.bankIdRestTypes.OrderResponse;
import com.hedvig.memberservice.enteties.SignStatus;
import lombok.Value;
import org.springframework.lang.NonNull;

@Value
public class MemberSignResponse {

  @NonNull
  SignStatus status;

  @NonNull
  OrderResponse bankIdOrderResponse;

}

File: member-service/src/main/java/com/hedvig/memberservice/services/member/dto/MemberSignResponse.kt
package com.hedvig.memberservice.services.member.dto

import com.hedvig.external.bankID.bankIdTypes.OrderResponse
import com.hedvig.memberservice.entities.SignStatus

data class MemberSignResponse(
    val signId: Long? = null,
    val status: SignStatus,
    val bankIdOrderResponse: OrderResponse? = null
)
